config/config.json

```diff
@@ -1,7 +1,7 @@
 {
-  "GCLOUD_PROJECT": "google.com:gulliver",
```

Comment:
Did you mean to change these?

I override these via environment variables, that are set by [direnv](http://direnv.net/). (So you have an `.envrc` file that lists the environment variables I want sent, and then hook into the shell so that whenever I change into the gulliver directory they are sent, and whenever I change into some other directory they are unset. You can also source the "parent" `.envrc` to set `GIT_AUTHOR_EMAIL`, etc. Works quite well.)

An alternative might be to use git's [assume aliases](http://durdn.com/blog/2012/11/22/must-have-git-aliases-advanced-examples/#assume-aliases) to mark files you don't want to commit.